src/Library/demos/Launcher/main.blp

```diff
@@ -0,0 +1,94 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Launcher";
+  description: _("Present a dialog of apps required to open a File/Uri");
```

Comment:
```suggestion
  description: _("Collect the arguments that are needed to open a File or URI");
```

The documentation says what happens depends on the desktop environment and might not open a dialog, so I think this is a bit better.